src/Search.php

```diff
@@ -62,7 +62,9 @@ public function __construct(Engine $engine, Router $router)
 	 */
 	public function search(string $query)
 	{
-		Debugger::timer('search_request');
+		if(class_exists(Debugger::class)) {
```

Comment:
Codestyle should be:

```suggestion
		if (class_exists('\Tracy\Debugger') === true) {
```

Reply:
@janbarasek Why `=== true`? the return type is not nullable bool

Reply:
It's only my codestyle. I think you can keep your version, it does not matter.

Thanks.